src/client/java/dev/creesch/WebInterface.java

```diff
@@ -42,6 +42,9 @@ public class WebInterface {
         "[\\n\\r§\u00A7\\u0000-\\u001F\\u200B-\\u200F\\u2028-\\u202F]"
     );
     private static final Pattern MULTIPLE_SPACES = Pattern.compile("\\s{2,}");
+    private static final Pattern SUPPORTED_COMMANDS = Pattern.compile(
+        "^/(msg|tell|w|me)(\\s|$)"
```

Comment:
I think you broke the PR with the latest commit switching to this regex. It no longer matches on more than just the command. 

Changing the regex to `"^/(msg|tell|w|me)(\\s.*)?$"` works for me.

Reply:
My bad.

Using `^/(msg|tell|w|me)(\\s.*|$)` instead - reads better to me for some reason.